src/Bundle/test/src/Tests/Controller/SubscriptionUiTest.php

```diff
@@ -137,6 +137,22 @@ public function it_allows_deleting_a_subscription(): void
         $this->assertEmpty($subscriptions);
     }
 
+    /** @test */
+    public function it_allows_deleting_multiple_subscriptions(): void
+    {
+        $this->loadFixturesFromFile('subscriptions.yml');
+
+        $this->client->request('GET', '/admin/subscriptions');
+        $this->client->submitForm('Bulk delete');
+
+        $this->assertResponseRedirects(null, expectedCode: Response::HTTP_FOUND);
```

Comment:
Is this test correct?

Reply:
Yes, It checks that the redirect is ok. (it's a code from @Zales0123 on ScienceBookUiTest)